service/transfer.go

```diff
@@ -0,0 +1,74 @@
+package service
+
+import (
+	"atmcase/model"
+	"atmcase/utils"
+	"fmt"
+	"strconv"
+
+	"github.com/shopspring/decimal"
+)
+
+// proses transfer uang
+func TransferUang(accountPengirim *model.AccountBank) {
+	fmt.Println("\nTransfer :")
+	fmt.Println("Masukan nomor rekening tujuan :")
+	strNorek, _ := utils.InputScan()
+	intNorek, err := strconv.Atoi(strNorek)
+	if err != nil {
+		fmt.Printf("\n>>>>>> Masukan Salah ! <<<<<<")
+		return
+	}
+	if check, accountPenerima := findAccount(intNorek); check {

```

Comment:
supaya if statementnya ngga terlalu panjang bisa dibuat seperti ini mas
```suggestion
        check, accountPenerima := findAccount(intNorek)
        if !check  {
             fmt.Println("\n>>> Akun tidak ditemukan ! <<<")
        }
```